### DISCUSSION TITLE
Load counts for all tree nodes by default

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5937] [Feature Requests]
Thanks to the tremendous work done in `v7.9.2` using recursive CTE with tree queries, counts load nearly instantaneously. Not only does it dramatically improve finding items in a tree, it also is a technically impressive feat that has excited every user we've demonstrated it to.

RBGE has tested this extensively and has been using this on their live deployment for months without any issues.

In the Remote User Prefs file now, users have to specify the Rank ID of the tree level they want to show counts at and below: 
`TreeEditor.Rank.Threshold.Taxon=140`

The feature to display only relevant nodes in the tree (https://github.com/specify/specify7/pull/4023) is limited until all counts for the entire tree are loaded. Currently, you can only view relevant nodes at levels with loaded counts, typically much lower in the tree than would be desired.

It would be incredible to load the counts for all nodes in a tree by default. It would be fine to add a preference if desired, but it certainly would be nice to have this as the default.

**Comment by grantfitzsimmons** [comment]
From @realVinayak:
> ^just a caution that I disabled CTE for paleo trees because I didn't have time to understand them, and to check if CTE is applicable there. Would need to check that. Right now, it is enabled for taxa, geography and storage

**Comment by realVinayak** [comment]
After some thoughts, this would bode well via a solution of https://github.com/specify/specify7/issues/3169


For reference, this determines whether CTE logic gets run: https://github.com/specify/specify7/blob/b03337126a8db91a9e6b5fe3dad0060bf0dfff55/specifyweb/specify/tree_views.py#L175

If that property is true, it first tries CTE. If underlying DB doesn't support it (say Mysql 5.7), it uses node number method (an optimization over previous queries)
